### PR TITLE
Feature/loaders

### PR DESCRIPTION
**Pull request by Gmarvis**

### CODE REVIEW

File: src/app/discussions/[id]/page.tsx [hunk 1]
```diff
@@ -260,14 +260,15 @@ const Chats = () => {
...
             onClose={() => setShowInfoCard((prev) => !prev)}
             picture={
+              receiver?.image ||
               "https://i.pinimg.com/564x/fe/85/c3/fe85c35b97c3f14082ac2edfb25eba44.jpg"
             }
-            name={"Caleb matins"}
+            name={receiver?.name}
             about={"made of gold"}
-            email={"calebmatins@gmail.com"}
+            email={receiver?.email}
```
Comment (GaelTikeng) [inline src/app/discussions/[id]/page.tsx:271]: receiver object does not have email

File: src/components/atoms/pulseLoader.tsx [hunk 2]
```diff
@@ -3,7 +3,7 @@ import React from "react";
 const PulseLoader = () => {
   return (
     <div className="pulsating-circle w-full my-auto">
-      <span className=" transform -translate-x-50 text-black"> Loding...</span>
+      <span className=" transform -translate-x-50 text-white"> Loding...</span>
```
Comment (GaelTikeng) [inline src/components/atoms/pulseLoader.tsx:6]: Loading
and 
not 
loding

File: src/components/organisms/ContactInfo.tsx [hunk 3]
```diff
@@ -37,9 +46,13 @@ const ContactInfo = ({
   const [openCard, setOpenCard] = useState(false);
   const [showAddMembers, setShowAddMembers] = useState(false);
   const router = useRouter();
-  const activeChat = JSON.parse(localStorage.getItem("activeChat") || "{}");
+  const [activeChat, setAtiveChat] = useState<Room>(
+    JSON.parse(localStorage.getItem("receiver") || "[]")
```
Comment (GaelTikeng) [inline src/components/organisms/ContactInfo.tsx:50]: change the name of this state to receiver

File: src/components/organisms/ContactInfo.tsx [hunk 4]
```diff
@@ -75,6 +88,34 @@ const ContactInfo = ({
...
+    setMembers((prev) => [...prev, ...selectedMember]);
+
+    LOCAL_STORAGE.save("group_members", [...members, ...selectedMember]);
+  };
+
+  // handle remove member
+  const HandleRemoveMember = (id: string) => {
+    const filteredMembers = members.filter((member) => member.id !== id);
+    setMembers(filteredMembers);
+    LOCAL_STORAGE.save("group_members", filteredMembers);
```
Comment (GaelTikeng) [inline src/components/organisms/ContactInfo.tsx]: I prefere you avoid using this coz at times it does not
 work

File: src/components/organisms/ContactInfo.tsx [hunk 5]
```diff
@@ -92,7 +133,7 @@ const ContactInfo = ({
           </div>
         </div>
 
-        {!isGroup ? (
+        {!activeChat.isGroup ? (
```
Comment (GaelTikeng) [inline src/components/organisms/ContactInfo.tsx]: change this to receiver

**Review by GaelTikeng — CHANGES_REQUESTED** [review]

File: src/app/discussions/[id]/page.tsx (hunk 1 above)
Comment (Gmarvis) [reply src/app/discussions/[id]/page.tsx:271]: and i just noticed the is no email again on the local storage

**Review by GaelTikeng — DISMISSED** [review]

**Review by GaelTikeng — APPROVED** [review]

**Pull request merged**